scrc/utils/sql_select_utils.py

```diff
@@ -273,14 +273,14 @@ def clean(judgments):
 
 # according to BFS: https://en.wikipedia.org/wiki/Subdivisions_of_Switzerland
 regions = {
-    "Eastern_Switzerland": ["SG", "TG", "AI", "AR", "GL", "SH", "GR"],
-    "Zürich": ["ZH"],
-    "Central_Switzerland": ["UR", "SZ", "OW", "NW", "LU", "ZG"],
-    "Northwestern_Switzerland": ["BS", "BL", "AG"],
-    "Espace_Mittelland": ["BE", "SO", "FR", "NE", "JU"],
-    "Région lémanique": ["GE", "VD", "VS"],
-    "Ticino": ["TI"],
-    "Federation": ["CH"],  # this is a hack to map CH to a region too
+    "Eastern_Switzerland": [17, 22, 3, 4, 10, 18, 11],
+    "Zürich": [27],
+    "Central_Switzerland": [23, 20, 16, 15, 13, 26],
+    "Northwestern_Switzerland": [7, 6, 2],
+    "Espace_Mittelland": [5, 19, 8, 14, 12],
+    "Région lémanique": [9, 24, 25],
+    "Ticino": [21],
+    "Federation": [1],  # this is a hack to map CH to a region too
 }
```

Comment:
Das ist nicht mehr so gut verständlich. Gäbe es eine Möglichkeit, dies verständlich zu behalten (die Kantonsabkürzungen bleiben erhalten)?